src/api/mod.rs

```diff
@@ -459,8 +459,8 @@ pub struct Config {
   pub threads: usize,
 }
 
-fn check_tile_log2(n: usize) -> bool {
-  ((1 << TilingInfo::tile_log2(1, n)) - n) == 0 || n == 0
+fn check_tile_log2(n: usize) -> Option<bool> {
+  Some(((1 << TilingInfo::tile_log2(1, n)?) - n) == 0 || n == 0)
```

Comment:
What is the distinction between `None` and `Some(false)` here? If they mean the same thing, I'd suggest keeping this a `bool` and returning `false` instead of `None`.

Reply:
Good point